README.md

```diff
@@ -1,19 +1,34 @@
 ![Integration Test](https://github.com/Azure/aml-compute/workflows/Integration%20Test/badge.svg?branch=master&event=push)
 ![Lint and Test](https://github.com/Azure/aml-compute/workflows/Lint%20and%20Test/badge.svg?branch=master&event=push)
 
-# Azure Machine Learning Compute Action
+# GitHub Action for creating compute for Azure Machine Learning 
 
 ## Usage
 
-The Azure Machine Learning Compute action will allow you to create a new compute target or check whether the specified compute target is available so you can later run your Machine Learning experiments or deploy your models remotely. If the compute target exists, it will just connect to it, otherwise the action can create a new compute target based on the provided parameters. Currently, the action only supports Azure ML Clusters and AKS Clusters. You will need to have azure credentials that allow you to create and/or connect to a workspace.
+The actions for creating compute for Azure Machine Learning will allow you to create a new compute target on [Azure Machine Learning](https://azure.microsoft.com/en-us/services/machine-learning/) using GitHub Actions.
 
-This action requires an AML workspace to be created or attached to via the [aml-workspace](https://github.com/Azure/aml-workspace) action.
+Get started today with a [free Azure account](https://azure.com/free/open-source)!
 
-## Template repositories
+This repository contains GitHub Action for creating and connecting to compute for Azure Machine Learning, so you can later train your machine learning models or deploy your models remotely. If the compute target exists, it will just connect to it, otherwise the action can create a new compute target based on the provided parameters. Currently, the action only supports Azure ML Clusters and AKS Clusters. 
+
+
+## Dependencies on other GitHub Actions

```

Comment:
Leaving the heading as is for now. Will coordinate offline and update this for all AzML actions